src/com/owncloud/android/utils/DisplayUtils.java

```diff
@@ -275,4 +284,52 @@ public static void colorSnackbar(Context context, Snackbar snackbar) {
         // Changing action button text color
         snackbar.setActionTextColor(ContextCompat.getColor(context, R.color.white));
     }
+
+    /**
+     * fetches and sets the avatar of the current account in the drawer in case the drawer is available.
+     *
+     * @param account        the account to be set in the drawer
+     * @param userIcon       the image view to set the avatar on
+     * @param avatarRadius   the avatar radius
+     * @param resources      reference for density information
+     * @param storageManager reference for caching purposes
+     */
+    public static void setAvatar(Account account, ImageView userIcon, float avatarRadius, Resources resources,

```

Comment:
Yes I know :disappointed: but I didn't find an easy way on how to achieve that... :(